### ISSUE TITLE
Problems Running Test Serverless Function

### ISSUE THREAD

**Issue by hassanbk20** [gwsystems/sledge-serverless-framework#379]
Hello we are a Senior Design Class at the University of Michigan using sledge for our ADAS application. We are trying to run our first test function on our raspberry pi 4, but the folder for .sledgert does not appear in our directory. Any ideas?

**Comment by hassanbk20** [comment]
I appreciate you getting back to us! We are trying to run the fibonacci test function, and on the command LD_LIBRARY_PATH="$(pwd):$LD_LIBRARY_PATH" ./sledgert ../../tests/fibonacci/bimodal/spec.json we get a command line error stating the folder sledgert does not exist. I can send screenshots for more context if needed.

**Comment by emil916** [comment]
`./sledgert` is not a directory, it is the path to the executable in the `runtime/bin` directory. You most likely have not compiled the runtime code. Follow the installation instructions listed on the repo's main page.

**Comment by hassanbk20** [comment]
After going through the steps again, we found that for ./install_deb.sh , we are getting "ARM64 support is still a work in progress!" . Could this be potentially be why we weren't able to run the test function?

**Comment by emil916** [comment]
Yes, indeed. For the time-being if you have a x86 machine, try installing on that one. Also, please share the error screenshot so we have it in the records. Thanks!
